Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python
+
+WORKDIR /opt/service
+
+RUN pip install poetry
+RUN poetry install

```

Comment:
Poetry cannot install anything (there is no pyproject.toml and poetry.lock at this step)

Reply:
Also, let's make poetry not create virtualenv inside our docker image

Reply:
virtualenv thing stil was not solved